### PR TITLE
Use new Next app directory

### PR DESCRIPTION
**Pull request by alan-hadyk**

### CODE REVIEW

File: src/components/molecules/Button/styles.ts [hunk 1]
```diff
@@ -38,22 +38,12 @@ export const buttonDefaultThemeClasses: IThemeClasses = {
...
-  pseudoClasses: [
-    "childrenRipple:animate-ripple-slow",
-    "childrenRipple:h-24",
-    "childrenRipple:w-24",
-    "childrenRipple:rounded-full",
-    "childrenRipple:ml-negative12 ",
-    "childrenRipple:mt-negative12",
-    "childrenRipple:opacity-0",
-    "childrenRipple:absolute"
-  ],
```
Comment (angulawk) [inline src/components/molecules/Button/styles.ts:50]: Is it no longer necessary?

**Comment by angulawk** [comment]
On screen width 1330px Tech Stack and Code are not fully visible:

ah-app

File: package.json [hunk 2]
```diff
@@ -17,52 +18,50 @@
...
     "@react-hook/resize-observer": "^1.2.6",
-    "axios": "^1.3.5",
+    "axios": "^1.4.0",
     "detect-browser": "^5.3.0",
     "lodash": "^4.17.21",
-    "next": "^13.2.4",
-    "puppeteer": "^19.4.1",
-    "react": "^18.2.0",
+    "next": "13.4.1",
+    "puppeteer": "^20.1.1",
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply package.json:25]: Yes, in `src/helpers/browser/convertPageToPDF.ts`

image

File: package.json [hunk 3]
```diff
@@ -17,52 +18,50 @@
...
     "isomorphic-fetch": "^3.0.0",
     "jest": "^29.5.0",
-    "jest-environment-jsdom": "^29.0.3",
-    "postcss": "^8.4.21",
-    "prettier": "^2.8.7",
-    "tailwindcss": "^3.3.2",
+    "jest-environment-jsdom": "^29.5.0",
+    "postcss": "8.4.23",
+    "prettier": "^2.8.8",
+    "tailwind-clip-path": "^1.0.0",
```
[the remark this answers is not in the document]
Comment (alan-hadyk) [reply package.json:58]: Yes, in `tailwind.config.ts`

**Comment by alan-hadyk** [comment]
> On screen width 1330px Tech Stack and Code are not fully visible:
> 
> ah-app

Fixed:

Zrzut ekranu 2023-05-18 o 21 43 00

**Review by angulawk — APPROVED** [review]

**Pull request merged**